sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci

```diff
@@ -949,7 +949,7 @@ trace
 Trace Statistics:
   SELECT (time: ?, fetch: ?, ioread: ?)
     SCAN (temp time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
-      SCAN (hash temp(h), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
+      SCAN (hash temp(f), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
```

Comment:
It is necessary to check whether the answer sheet should be changed or the test case should be changed.

Reply:
before was setting system parameter 'max_hash_list_scan_size=4M'.
now default increased to 8M.
does that mean size minimum value starts from 8M? if yes test case have to be modified.